helm-chart-sources/pulsar/templates/proxy/proxy-configmap.yaml

```diff
@@ -122,3 +122,15 @@ data:
   PULSAR_MEM: {{ .Values.proxy.configData.PULSAR_MEM }}
   PULSAR_GC: {{ .Values.proxy.configData.PULSAR_GC }}
 {{- end }}
+{{- if .Values.proxy.kafkaOnPulsarEnabled }}
+  # enabling Kafka-on-Pulsar
+  proxyProtocolHandlerDirectory: "./proxyprotocols"
+  proxyMessagingProtocols: "kafka"  
```

Comment:
It should probably be a collection where we add the messaging protocol. So we can add multiple protocols (still thinking about RabbitMQ)

Reply:
this is for "{{- if .Values.proxy.kafkaOnPulsarEnabled }}"

you are right.
btw you can configure directly proxyMessagingProtocols in the values.yaml file

this is a short cut to make things simpler for KOP

but I agree with you, a more general solution is better

Reply:
My main concern about a short cut is that once it is released, we need to maintain backwards compatibility, and that can make helm charts harder to read and harder to develop.